project/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -98,4 +115,15 @@ private static LatLng getLatLngFromString(String coordinates) {
     String[] latLng = coordinates.split(",");
     return new LatLng(Double.parseDouble(latLng[0]), Double.parseDouble(latLng[1]));
   }
+
+  // Store the user's preferences in the database, only if the user is signed in an registered.
```

Comment:
I think there's a typo - "and" instead of "an".
I think it's not accurate to say that this happens if the user is signed in *and registered*, because you don't check if they're registered (you assume that, which I think is fine).

Reply:
Sure, I changed it